docs/extensions.rst

```diff
@@ -0,0 +1,48 @@
+Extension Modules
+==================
+
+The below modules are Tripal 3 compatible and grouped roughly by category:
+
+.. toctree::
+   :maxdepth: 2
+
+   extensions/administrative
+   extensions/analysis
+   extensions/datainput
```

Comment:
in the rest of the guide, we use snake case.  let's do so here as well for consistency.